webapp/src/pages/reviewForm/components/ReviewForm.js

```diff
@@ -200,7 +200,7 @@ const ReviewForm = (props) => {
             isOpen: props.event.is_review_open,
             eventId: event.event.id,
             stage: currentStage,
-            deadline: event.event.review_close,
+            deadline: event.event.review_close==null?"2099-12-05T23:59:59":event.event.review_close,
```

Comment:
You can do this slightly more concisely (and actually more correctly because it also caters for other kinds of null-like values) by using the null coalescing operator instead: 
`event.event.review_close ?? "2099-12-05T23:59:59"`